lighthouse-cli/test/smokehouse/lighthouse-runners/devtools.js

```diff
@@ -10,60 +10,44 @@
 
 import fs from 'fs';
 import os from 'os';
-import {spawn} from 'child_process';
+import {execFile} from 'child_process';
```

Comment:
Would we lose anything by using execFileSync? Seems no, since we wrap all usages of it in a promise and await anyway.

Reply:
`execFileSync` is actually better sounds good